packages/ui-components/src/components/form/TextInput.vue

```diff
@@ -305,13 +306,15 @@ const sizeClasses = computed((): string => {
   switch (props.size) {
     case 'sm':
       return 'h-6 text-sm'
+    case 'md':
+      return 'h-8 text-sm'
```

Comment:
The difference between md and base is negligible, do we really want to add extra complexity here by adding this new size?

Reply:
This was added so the heights would be equal here:
<img width="1242" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/748a73b2-c3b4-470c-9844-9433a9c23b12">

Previously, there was a different Select style to TextInput. One used borders and the other outlines and shadows. 

I added a new button style called "foundation" to match the select style. This has an unwanted side effect where it now was a different height than the select inputs, due to the difference in border and outline. 

I added this new size to match the select inputs height. I couldn't just update base as non "foundation" inputs would be affected.

Reply:
@andrewwallacespeckle Maybe it's better to just ensure the existing button & select styles are the same height? Sounds like we're adding extra complexity not even cause there's a design need, but to deal with existing styles being inconsistent